bedrock/settings/base.py

```diff
@@ -1790,13 +1790,11 @@ def before_send(event, hint):
     "IM",  # Isle of Man (a British Crown dependency)
     "IO",  # British Indian Ocean Territory
     "JE",  # Jersey (a British Crown dependency)
-    "UK",  # United Kingdom
```

Comment:
Note for reviewer: the new tests added here have uncovered that `UK` and `UM` are in fact unsupported country codes, and have never in fact been returning in our requests. Both of these pre-date VPN coming to bedrock, so I think can safely be removed. According to https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements, `UK` is simply reserved by the United Kingdom, it's not actually used (`GB` is the official code).

Reply:
PS: Next time someone edits the list or the whereabouts, the GB comment would also use some love:

> "GB",  # United Kingdom of Great Britain and Northern **Island**

(unless that's an inside joke I'm not following;D…)